tests/backup/backup_test.go

```diff
@@ -187,7 +187,7 @@ var _ = Describe("{BackupCreateKillStoreRestore}", func() {
 			}
 			// TODO(stgleb): Adjust this logic to skip cluster scoped resources
 			// that do not get backed up
-			ValidateApplications(contexts)
```

Comment:
Why don't we add timeout to context as a field? Maybe even add context.Context as a field to control timeout and cancellation.

Reply:
Changed as per comment